src/main/scala/mesosphere/marathon/core/launcher/impl/TaskLabels.scala

```diff
@@ -34,7 +34,7 @@ object TaskLabels {
     ReservationLabels(Map(
       FRAMEWORK_ID_LABEL -> frameworkId.id,
       // This uses taskId.reservationId to match against the id that was originally used to create the reservation
-      TASK_ID_LABEL -> Task.Id.reservationId(taskId.idString)
+      TASK_ID_LABEL -> taskId.idString //Task.Id.reservationId(taskId.idString)
```

Comment:
This is an important change. I don't think it is currently correct. What should reservation ids look like?